client/post-editor/editor-publish-button/test/index.jsx

```diff
@@ -0,0 +1,376 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import moment from 'moment';
+import React from 'react';
+import sinon from 'sinon';
+import TestUtils from 'react-addons-test-utils';
+import identity from 'lodash/identity';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+/**
+ * Module variables
+ */
+
+describe( 'EditorPublishButton', function() {
+	var EditorPublishButton,
+		shallow,
+		MOCK_SITE = {
+			capabilities: {
+				publish_posts: true
+			},
+			options: {}
+		};
+
+	useFakeDom();
+
+	this.timeout( 10 * 1000 );
+
+	before( function() {
+		shallow = require( 'enzyme' ).shallow;
```

Comment:
Copy & paste from another test suite. I realize now that's not needed, I've changed it to an `import` on the top of the file.